tests/unit_tests/parameters/test_serial_parameters.py

```diff
@@ -0,0 +1,117 @@
+from pathlib import Path
+
+import pytest
+
+from i19_bluesky.parameters.components import PandaRotationParams, ZebraRotationParams
+from i19_bluesky.parameters.serial_parameters import (
+    GridParameters,
+    GridType,
+    SerialExperiment,
+    SerialExperimentEh2,
+    WellsSelection,
+)
+
+
+@pytest.fixture
+def dummy_wells_settings():
+    return {
+        "first": 0,
+        "last": 5,
+        "selected": [1, 3, 5],
+        "series_length": 3,
+        "manual_selection_enabled": True,
+    }
+
+
+@pytest.fixture
+def dummy_serial_params(dummy_wells_settings):
+    return {
+        "hutch": "EH2",
+        "visit": "/tmp/i19-2/cm12345-1",
+        "dataset": "foo",
+        "filename_prefix": "bar_01",
+        "images_per_well": 10,
+        "exposure_time_s": 0.1,
+        "image_width_deg": 0.1,
+        "detector_distance_mm": 320,
+        "two_theta_deg": 0,
+        "transmission_fraction": 0.3,
+        "grid": {
+            "grid_type": "Silicon",
+            "x_steps": 20,
+            "z_steps": 20,
+        },
+        "wells": dummy_wells_settings,
+        "rot_axis_start": -5,
+        "rot_axis_increment": 0.1,
```

Comment:
Could: Have 'rotation' instead of 'rot' just for clarity
```suggestion
        "rotation_axis_start": -5,
        "rotation_axis_increment": 0.1,
```